recipe/meta.yaml

```
@@ -22,7 +22,7 @@ requirements:
     - pip
     - python >=3.6
   run:
-    - ipywidgets >=7.6.0,<8.0.0
+    - ipywidgets >=7.6.0
     - numpy
     - python >=3.6
```

```suggestion
    - python >=3.9
```

Latest jupyter-rfb requires >=3.9

---

Only for 0.4.1, which is #6.

---

ah, I didn't notice this is the recipe for v0.3